REPOSITORIES.md

```diff
@@ -0,0 +1,80 @@
+## Requirements for Motorcycle.js Repositories
+
+This document aims to normalize and maintain high quality
+repositories for the Motorcycle.js project.
+
+### Table of Contents
+- [Testing](#testing)
+  - [Preferred Tools](#preferred-testing-tools)
+  - [Continuous Integration](#continuous-integration)
+- [Code Style](#code-style)
+- [NPM Scripts](#npm-scripts)
+- [Dependencies](#dependencies)
+- [Github Usage](#github-usage)
+
+#### Testing
+
+All repositories included in, or to be considered for inclusion, the
+Motorcycle.js organization **must** have proper tests, which cover at
+least basic usage of the library in question.
+
+###### Preferred Testing Tools
+Although not a requirement, these tools are recommended for keeping
+the tooling the same across Motorcycle.js repositories.
+
+- [EsLint](http://eslint.org/) - [Code Style](#code-style) Checking
+- [Mocha](https://mochajs.org/) - Test Framework
+- [Testem](https://github.com/testem/testem) - Test Runner
+- [Assert](https://github.com/defunctzombie/commonjs-assert) - Assertions
+
+###### Continuous Integration
+Every repository **must** enable continuous integration through
+[Travis-CI](https://travis-ci.org/) with passing builds before
+consideration of being included into the motorcyclejs Github
+organization.
+
+Here is a [sample configuration](example-travis.yml) for Travis-CI.
+
+#### Code Style
+
+All repositories included in Motorcyclejs **must** use the following to
+adhere to code style standards:
+- [eslint](http://eslint.org)
+- [eslint-config-cycle](https://github.com/cyclejs/eslint-config-cycle)
+- [eslint-plugin-cycle](https://github.com/cyclejs/eslint-plugin-cycle)
+- [eslint-plugin-no-class](https://github.com/emmenko/eslint-plugin-no-class)
+
+Optionally:
+- [babel-eslint](https://github.com/babel/babel-eslint) - If your
+project is using ES6 or Babel for transpilation.
+
+#### NPM Scripts
+Required:
+- `eslint` - for running eslint checks
+- `test` - for running tests
+- `start` - must be `npm install && npm prune && validate-commit-msg`
```

Comment:
If this should work, there’s also a requirement for the dev dependency `validate-commit-msg`.